scripts/PSRA_copyTables_Canada.py

```diff
@@ -0,0 +1,302 @@
+# Import statements
+import argparse
+import configparser
+import csv
+import logging
+import os
+import sys
+'''
+Script to copy Canada PSRA tables into PostGIS
+python3 PSRA_copyTables_Canada.py
+'''
+
+
+# Main Function
+def main():
+    logging.basicConfig(level=logging.INFO,
+                        format='%(asctime)s - %(levelname)s - %(message)s',
+                        handlers=[logging.FileHandler('/tmp/{}.log'.format(
+                            os.path.splitext(sys.argv[0])[0])),
+                            logging.StreamHandler()])
+    os.chdir(sys.path[0])
+    args = parse_args()
+
+    # Copy agg curves stats b0 table
+    with open("/usr/src/app/ebRisk/Canada/ebR_Canada_agg_curves-stats_b0.csv".format(
```

Comment:
We should avoid system paths.

Reply:
Pass the path in as an argument?

Reply:
Probably best.

Reply:
Or maybe get the path from an environment variable, and with a default fallback if the environment variable is unset?
(Or maybe accept both command-line argument and environment variable?)

I took a brief look at it back in June 2021, and from what I recall, it is not just the Python scripts, but also some SQL files where the `/usr/src/app` path is hardcoded.  See OpenDRR/opendrr-api#114, and the newly opened OpenDRR/model-factory#98.